endpoints.json

```diff
@@ -25,5 +25,24 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/article_id": {
+    "description": "serves an object of a article",
+    "queries": ["article_id"],
```

Comment:
the article_id is not a query in this implementation, what indicates a query and how do we access those? is the article_id being accessed in this way?